Changes to docs to reflect current behavior with respect to licensing (#8000)
solo-io/gloo: docs/content/operations/updating_license/_index.md

## Confirm that your license expired

> Whether you're a prospective user using a trial license or a full Gloo Edge subscriber, this license can expire. When it does, you may see warnings on your Gloo Edge deployments that are new to you.

Reviewer: can you double check this? i believe some pods still do crash (e.g. observability, gloo-fed...) whereas some will work fine but log a warning

Whether you're a prospective user using a trial license or a full Gloo Edge subscriber, this license can expire. When it does, you may see warnings on your Gloo Edge deployments that are new to you.

For example, from the following [k9s](https://k9scli.io/) display, you can see that certain `gloo-system` pods fall into a `CrashLoopBackoff` state.

> ![k9s Display with Expired License]({{% versioned_link_path fromRoot="/img/k9s-license-expired.png" %}})

Reviewer: we can also delete the image `k9s-license-expired.png` from the repo

![k9s Display with Refreshed License]({{% versioned_link_path fromRoot="/img/k9s-license-refreshed.png" %}})

Congratulations! You have successfully replaced your Gloo Edge Enterprise license key.

> ## Outdated License format

Reviewer 2: Should this section be named something more like "Expired GraphQL license add-on"?

> ## Expired GraphQL license add-on

Reviewer: ```suggestion
## Older format licenses
```
this is not exactly referring to expired licenses, but rather licenses that were created in an older format (but are still valid)

Reviewer: not sure that a good title for this section is, maybe "deprecated license format"?

> In some cases, Gloo Edge users who are have GraphQL features enabled may see the following message

Reviewer 2: ```suggestion
If you enabled GraphQL features, you might see the following log on the `observability` deployment:
```

```bash
% kubectl logs deploy/observability -n gloo-system
{"level":"warn","ts":"2023-03-23T18:56:48.554Z","logger":"observability","caller":"client/client.go:195","msg":"Your gloo license graphql addon is outdated. Please contact support to update your license.","version":"1.14.0-beta10"}
```

> In this case, contact support to get a new license key.

Reviewer: ```suggestion
Please note that Gloo Edge will still function correctly, as we accept both the new and deprecated license formats. However, to get rid of the warning in the logs, you should contact support to get a new license key.
```

> If this message appears please contact support to get an up-to-date license key.

Reviewer 2: ```suggestion
In this case, contact support to get a new license key.
```
